launcher_scripts/nemo_launcher/core/data_curation_stages.py

```diff
@@ -81,6 +81,7 @@ def _make_cluster_parameters(self, cluster: str) -> Dict:
         job_name = run_cfg.get("name")
         time_limit = run_cfg.get("time_limit")
         nodes = run_cfg.get("nodes")
+        dependency = run_cfg.get("dependency")
```

Comment:
Ryan Wolf
  [7:11 AM](https://nvidia.slack.com/archives/D068BCN29C0/p1704294691958209)
The dependency argument is just used for making sure the stages and sub-stages run in order. It was accidentally removed for the data curation substages in a previous commit.